docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,375 @@
+# How to use go-plugin-runner in APISIX Ingress
+
+## Proposal Description
+
+Based on version 0.3 of the go-plugin-runner plugin and version 1.4.0 of APISIX Ingress, this article goes through building the cluster, building the image, customizing the helm chart package, and finally, deploying the resources. It is guaranteed that the final result can be derived in full based on this documentation.
```

Comment:
Would it be better to express the different steps in a list?

Reply:
Solved~